src/Content/FormUrlEncodedStringContent.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <copyright file="FormUrlEncodedStringContent.cs" company="ProjectBlackmagic">
+// RestfulClient
+// Copyright (c) ProjectBlackmagic. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Github: https://github.com/ProjectBlackmagic/RestfulClient.
+// </copyright>
+
+using System.Net.Http;
+using System.Text;
+
+namespace ProjectBlackmagic.RestfulClient.Content
+{
+    /// <summary>
+    /// FormUrlEncodedStringContent defines a subtype of <see cref="StringContent"/> for JSON content.
+    /// </summary>
+    public class FormUrlEncodedStringContent : StringContent
+    {
+        private static readonly Encoding StringEncoding = Encoding.UTF8;
```

Comment:
I don't think we should make those string private, quite opposite, they should be public as you want to see them in your stack trace. Also, let's create an abstract Content class and make them part of it so every child should implement them.

Reply:
I need these to be ```static``` in order to use them when calling the base (```StringContent```) constructor (line 26), but I cannot define ```static abstract``` properties on the a new abstract content class. My only other choice is to define an interface.